src/app/shell/main/result/hobby-groups-list/hobby-groups-list.component.html

```diff
@@ -1 +1,4 @@
-<p>hobby-groups-list works!</p>
+<div fxLayoutAlign="left center">
+  <span>Знайдено Х пропозицій</span>
+  <app-ordering></app-ordering>
```

Comment:
let make follow single responsibility principles. crete two inputs: options and selectedOption so we can extract getting ordering state from this component